README.md

```diff
@@ -456,10 +456,15 @@ access to HTTP verbs `GET`, and `HEAD`. This also includes any kind of
 QUERY or SEARCH verbs, if supported.
 
 ##### `acl:Write`
-gives access to a class of operations that can modify the resource. In a REST
-API context, this would include `PUT`, `POST`, `DELETE` and `PATCH`. This also
+gives access to a class of operations that can modify an existing resource.
+There are two levels: write permissions on an existing resource allows modifying
+(but not deleting) that resource.
+In a REST API context, this would include `POST` and `PATCH`. This also
 includes the ability to perform SPARQL queries that perform updates, if those
 are supported.
+For creating or deleting a resource, write permissions on the container of which
+the resources is a member, are also needed.
```

Comment:
"also" implies in addition to something else, but that's not the case, right?

Reply:
ah, good catch. that's what i thought yes, that you need write permissions on both the container and the resource. is it only required on the container?

Reply:
Just the container, as far as I understand.

Reply:
ah, good catch. that's what i thought yes, that you need write permissions on both the container and the resource. is it only required on the container and not on the resource itself?

Reply:
ok, will double-check what you implemented in NSS, and describe that :)

Reply:
ah, it's "instead", indeed. https://github.com/solid/node-solid-server/pull/1014/files#diff-b11433f1fb19c85e170915d27502d490R27